### PR TITLE
Quicksilver 2

### PR DESCRIPTION
**Pull request by joe-bowman**
- archive quicksilver-1 genesis, README and gentxs
- add migration script
- update instructions

### CODE REVIEW

**Review by ajansari95 — APPROVED** [review]

File: quicksilver-1/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  go: go version go1.19 linux/amd64
+  ```
+## Genesis validators
+
+## Create a validator
+
+1. Init Chain and start your node
+
+   ```sh
+   > quicksilverd init <moniker-name> --chain-id=quicksilver-1
```
Comment (ajansari95) [inline quicksilver-1/README.md:51]:
```suggestion
   > quicksilverd init <moniker-name> --chain-id=quicksilver-2
```

File: quicksilver-1/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+   --amount 50000000uqck \
+   --commission-max-change-rate "0.1" \
+   --commission-max-rate "0.20" \
+   --commission-rate "0.1" \
+   --min-self-delegation "1" \
+   --details "a short description lives here" \
+   --pubkey=$(quicksilverd tendermint show-validator) \
+   --security-contact "youremail@goes.here" \
+   --moniker <your_moniker> \
+   --chain-id quicksilver-1 \
```
Comment (ajansari95) [inline quicksilver-1/README.md:95]:
```suggestion
   --chain-id quicksilver-2 \
```

**Review by ajansari95 — CHANGES_REQUESTED** [review]

File: quicksilver-1/README.md (hunk 1 above)
Comment (joe-bowman) [reply quicksilver-1/README.md]: This is in the quicksilver-1 repo; it's archived for posterity.

File: quicksilver-1/README.md (hunk 2 above)
Comment (joe-bowman) [reply quicksilver-1/README.md]: This is in the quicksilver-1 repo; it's archived for posterity.

**Review by ajansari95 — APPROVED** [review]

File: quicksilver-1/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+   --amount 50000000uqck \
+   --commission-max-change-rate "0.1" \
+   --commission-max-rate "0.20" \
+   --commission-rate "0.1" \
+   --min-self-delegation "1" \
+   --details "a short description lives here" \
+   --pubkey=$(quicksilverd tendermint show-validator) \
+   --security-contact "youremail@goes.here" \
+   --moniker <your_moniker> \
+   --chain-id quicksilver-2 \
```
Comment (ajansari95) [inline quicksilver-1/README.md:95]:
```suggestion
   --chain-id quicksilver-1 \
```

File: quicksilver-1/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  go: go version go1.19 linux/amd64
+  ```
+## Genesis validators
+
+## Create a validator
+
+1. Init Chain and start your node
+
+   ```sh
+   > quicksilverd init <moniker-name> --chain-id=quicksilver-2
```
Comment (ajansari95) [inline quicksilver-1/README.md:51]:
```suggestion
   > quicksilverd init <moniker-name> --chain-id=quicksilver-1
```

**Pull request merged**